docs/manual/comparison_with_sql.rst

```diff
@@ -0,0 +1,662 @@
+.. py:currentmodule:: datatable
+
+Comparison with SQL
+====================
+
+This page provides some examples of how various `SQL <https://en.wikipedia.org/wiki/SQL>`__ operations can be performed in ``datatable``. The ``datatable`` library is still growing; as such, not all functions in ``SQL`` can be replicated yet. If there is a feature you would love to have in ``datatable``, please make a feature request on the `github issues <https://github.com/h2oai/datatable/issues>`__ page.
```

Comment:
Do we want to mention which particular SQL implementation we're discussing in this tutorial?

Reply:
The code I used is pretty generic and applies to all. I used SQLite by the way

Reply:
Nice, if it works in SQLite it should work almost everywhere...